10.9-libcxx/stable/main/finkinfo/sci/nco.info

```diff
@@ -91,21 +92,24 @@ Replaces: <<
 	%N453-dev,
 	%N455-dev,
 	%N466-dev,
-	%N469-dev,
-	%N471-dev
+	%N469-dev
 <<
 
 # Unpack Phase:
 Source: https://github.com/nco/nco/archive/%v.tar.gz
 SourceRename: %n-%v.tar.gz
-Source-MD5: ae7640bb18b61bb86cde93ee70800087
-Source-Checksum: SHA1(070db5e0d4b966b1adc5f29074c3b9324bd540bd)
+Source-MD5: 84296a0ec44b9a58dab198fda9fac7f0
+Source-Checksum: SHA1(e39bdb3d50df0c7bb78c808e5b69907f9d2b3f75)
 
 # Patch Phase:
 PatchScript:  <<
 #!/bin/sh -ev
+	# Make sure to link against newly built libnco.dylib
 	perl -pi -e 's|-lnco|%b/src/nco/.libs/libnco.dylib|' src/nco++/Makefile.in
-	perl -pi -e "s/(apple.*)enable_shared='no'/\1/; s/(10\.\[012\])(\*)/\1\,.\2/" configure
+	# Patch configure to force enable_shared, since it suggests that Darwin cannot create shared linraries
+	perl -pi -e "s/(apple.*)enable_shared='no'/\1/" configure
+	# Patch configure to not link like Puma on Yosemite
+        perl -pi -e 's|10\.\[012\]\*|10.[012][,.]*|g' configure
```

Comment:
"not link like Puma on Yosemite"  What does this mean?

Reply:
Thanks for all the fixes and explanations!

"Puma" is OS X 10.1. There is a string-regexp bug in an old version of libtool: OS X version string "10.10" (10.10 is "Yosemite") and later versions all match "10.1", so it uses the type of linking that was correct on 10.1 rather than what is correct on modern platforms. The patch changes the string-regexp to match "10.1." instead. This was fixed long ago in libtool, but the upstream source distributors have to upgrade their libtool as part of their distribution process (it is statically included in the tarball, not automatic on the machine where one compiles).

Reply:
@schwehr: I hope the explanation of @dmacks is sufficient.